datasetinsights/data/datasets/synthetic.py

```diff
@@ -297,10 +340,74 @@ def exists(capture_file):
 
         return catalog[keep_mask]
 
-    def download(self, manifest_file):
+    def download_captures_from_manifest(self, manifest_file):
         """ Download captures of a given manifest file.
 
         Args:
             manifest_file (str): path to a manifest file
         """
         _download_captures(self.root, manifest_file)
+
+    @staticmethod
+    def download(data_root, version):
```

Comment:
Can you add unittest for this download method? Make sure it raises exceptions correctly. You can take `test_groceriesreal.py` as an example.